### PR TITLE
Milestone-2: Hit the API

### PR DESCRIPTION
**Pull request by Hope1226**
In this feature branch I:
 - Created module data.js;
 - Created a function to post new game in the API;
 - Created a function to get the data from the API;
 - Created a function to display the correct date in UI;
 - Integrated all modules in the index.js

### CODE REVIEW

File: src/app/data.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+export const displayData = async (id, conntainer) => {
+  const response = await fetch(`${api}/${id}/scores`);
+  const data = await response.json();
+  const scoreList = data.result;
+  const scoreBoard = document.createElement('div');
+  scoreBoard.classList.add('scores-container');
+  scoreList.forEach((obj) => {
+    const userName = obj.user;
+    const userScore = obj.score;
```
Comment (SajjadAhmad14) [inline src/app/data.js:44]: 1. User should be able to submit data and it reflects on the scoreboard. There are errors on the console and the app does not submit and receive data from API endpoints. Please make sure it runs as expected.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Hope Ustabaev, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/app/data.js (hunk 1 above)
Comment (Hope1226) [reply src/app/data.js:44]: Hi @SajjadAhmad14 , thanks for your comments, however, the app is working perfectly and as expected in my machine, even I tried to use it via git hub pages: 
[live demo](https://hope1226.github.io/Leaderboard/)
no issue found. have you tried to refresh after adding the score and name? what kind of errors is being returned in the console? I would appreciate it if you could be more specific. Thanks for your reply!

**Review by wandji20 — APPROVED** [review]
## Hi @Hope1226  :raised_hands:

##### You have done exceptionally well, and have met all requirements for this project :+1:

# Status: Approved 👍 :partying_face: 
Your project has been approved, however, Keep up the good work.

## Congratulations :+1:  :fireworks: 
## Optional
- N/A

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**